File: FRONTEND/javaScript/codigo-Js/edteam/algoritmos/busquedaMaxMin.js
function BusquedaMaxMin(arreglo){
    maximo = arreglo[0]
    minimo = arreglo[0]

    for(i=1;i<arreglo.length;i++){
        if(arreglo[i]>maximo){
            maximo = arreglo[i]
        }
        if(arreglo[i]<minimo){
            minimo = arreglo[i]
        }
    }
    return [maximo,minimo]
}

console.log(BusquedaMaxMin([0,1,3,7,3,9,4,3,6,10]))

File: BAKEND/python/codigo-Python/algoritmos/busquedaMaxMin.py
def busquedaMaxMin(arreglo):
    maximo = arreglo[0]
    minimo = arreglo[0]
    for i in range(1,len(arreglo)):
        if arreglo[i]>maximo:
            maximo = arreglo[i]
        if arreglo[i]<minimo:
            minimo = arreglo[i]
    return maximo,minimo

print(busquedaMaxMin([5,3,5,4,1,4,0,9]))